internal/batches/executor/changeset_specs.go

```diff
@@ -0,0 +1,215 @@
+package executor
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/sourcegraph/go-diff/diff"
+	"github.com/sourcegraph/src-cli/internal/batches"
+)
+
+func createChangesetSpecs(task *Task, result executionResult, features batches.FeatureFlags) ([]*batches.ChangesetSpec, error) {
```

Comment:
Nothing change here. Everything in this file is moved out of `executor.go`. It's all related to building/rendering changeset specs.